documentation.html

```diff
@@ -39,8 +42,208 @@ <h6>Components</h6>
+                        <p>Felix UI includes several predefined button styles, each serving its own semantic purpose.</p>
+                        <div class="element_container">
+                            <button class="btn btn--primary">Primary</button>
+                            <button class="btn btn--info">Info</button>
+                            <button class="btn btn--success">Success</button>
+                            <button class="btn btn--warning">Primary</button>
+                            <button class="btn btn--danger">Danger</button>
+                            <!-- <button class="btn btn--primary">Primary</button> -->
+                        </div>
+                        <pre>
+    <code class="html">
+<button class="btn btn--primary">Primary</button>
+<button class="btn btn--info">Info</button>
+<button class="btn btn--success">Success</button>
+<button class="btn btn--warning">Primary</button>
+<button class="btn btn--danger">Danger</button>
+    </code>
+</pre>
+                        <h4>Button variants</h4>
+                        <div class="element_container">
+                            <button class="btn btn--primary btn--round">Primary</button>
+                            <button class="btn btn--primary btn--outline">Primary</button>
+                            <button class="btn btn--primary btn--round btn--outline">Primary</button>
+                            <button class="btn btn--primary btn--ghost">Primary</button>
+                            <button class="btn btn--primary btn--link">Primary</button>
+                        </div>
+                        <pre>
+    <code class="html">
+<button class="btn btn--primary btn--round">Primary</button>
+<button class="btn btn--primary btn--outline">Primary</button>
+<button class="btn btn--primary btn--round btn--outline">Primary</button>
+<button class="btn btn--primary btn--ghost">Primary</button>
+    </code>
+</pre>
+                    </section>
+                </div>
             </div>
         </main>
+
+        <script src="//cdn.jsdelivr.net/gh/highlightjs/cdn-release@11.4.0/build/highlight.min.js"></script>
+        <script>
+            const sections = document.querySelectorAll("section")
+            const link = document.querySelectorAll(".link")
+            window.onscroll = () => {
+                var current = ""
+
+                sections.forEach((section) => {
+                    const sectionTop = section.offsetTop
+                    if (pageYOffset >= sectionTop - 200) {
+                        current = section.getAttribute("id")
+                    }
+                })
+
+                link.forEach((item) => {
+                    item.classList.remove("active")
+                    if (item.href.includes(current)) {
+                        item.classList.add("active")
+                    }
+                })
+            }
+            document.querySelectorAll("code").forEach(function (element) {
```

Comment:
can you `=>` here as well. just for consistency.

Reply:
Great observation. Thanks 😃